deploy-from-docker.md

```diff
@@ -0,0 +1,79 @@
+# Docker Support
```

Comment:
seems like you've overwritten the whole file somehow. Could you include only your changes?

Reply:
Of course. Sorry.